src/kbc_modules/sample_kbc/mod.rs

```diff
@@ -47,6 +47,20 @@ impl KbcInterface for SampleKbc {
 
         Ok(plain_text)
     }
+
+    fn get_resource(&mut self, description: String) -> Result<Vec<u8>> {
+        let desc: ResourceDescription =
+            serde_json::from_str::<ResourceDescription>(description.as_str())?;
+
+        match desc.name.as_str() {
+            "Policy" => Ok(std::include_str!("policy.json").as_bytes().to_vec()),
+            "Sigstore Config" => Ok(std::include_str!("sigstore_config.yaml")
+                .as_bytes()
+                .to_vec()),
+            "GPG Keyring" => Ok(std::include_str!("pubkey.gpg").as_bytes().to_vec()),
```

Comment:
Is there any way we can avoid hard coding the name of each resource inside of every KBC? Maybe we should have a shared enum for this?

Reply:
Done.